tests/Unit/User/Infrastructure/Command/ArrayCursor.php

```
@@ -0,0 +1,92 @@
+        throw new RuntimeException('Cursor id is not available in tests.');
+    }
+
+    #[\Override]
+    public function getServer(): Server
+    {
+        throw new RuntimeException('Cursor server is not available in tests.');
+    }
+
+    /** @param array<string, string|array<string, string>> $typemap */
+    #[\Override]
+    public function setTypeMap(array $typemap): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$typemap'. (undefined)

(UnusedFormalParameter)